src/Microsoft.DotNet.CodeFormatting/Rules/UsesXunitForTestsFormattingRule.cs

```diff
@@ -52,6 +52,23 @@ internal UsesXunitForTestsFormattingRule(Options options)
 
             foreach (var usingSyntax in root.Usings)
             {
+                Action<UsingDirectiveSyntax> addUsing = (UsingDirectiveSyntax newUsingSyntax) =>
```

Comment:
Sure.  I'll implement it as UsingDirectiveSyntax RemoveLeadingAndTrailingCompilerDirectives(this UsingDirectiveSyntax) and update the PR.